qual_id/categories/electronic.py

```diff
@@ -0,0 +1,17 @@
+from ..category import Category
+
+
+class Electronic(Category):
+  def get_values(self):
+    return [
+        'laptop',
+        'Desktop',
```

Comment:
```suggestion
        'desktop',
```
This should be lowercase